src/content/docs/fr/recipes/bun.mdx

```diff
@@ -90,7 +90,7 @@ bunx --bun astro preview
 
 ## Ajouter SSR avec Bun
 
-Puisque Bun offre la [compatibilité API Node.js](https://bun.sh/docs/runtime/nodejs-apis), vous pouvez utiliser n'importe quel adaptateur Astro pour le [rendu côté serveur](/fr/guides/on-demand-rendering/) dans votre projet Astro :
+Puisque Bun offre la [compatibilité API Node.js](https://bun.sh/docs/runtime/nodejs-apis), vous pouvez utiliser n'importe quel adaptateur Astro pour [rendre à la demande](/fr/guides/on-demand-rendering/) votre projet Astro :
```

Comment:
```suggestion
Puisque Bun offre la [compatibilité API Node.js](https://bun.sh/docs/runtime/nodejs-apis), vous pouvez utiliser n'importe quel adaptateur Astro pour [un rendu à la demande](/fr/guides/on-demand-rendering/) votre projet Astro :
```

Reply:
Maybe it's better? (Optionnal)

Reply:
For me it's the same thing, so if you prefer that's fine with me. However, you need to update your suggestion: we need `de` before `votre projet Astro` so that it remains French. 😉

Reply:
Ah yes, if you want commit the suggestion. For the rest it's good for me. About https://github.com/withastro/docs/pull/10350#discussion_r1882010893, you will create another PR when `api-reference` is edited?

Reply:
I don't think this is useful... It is the new link. But maybe we misunderstood each other. We have two options:
1. do not update the link, and remember to update it with `api-reference`... Since this is a code snippet and not a regular link, I'd say that we risk easily forgetting it...
2. update the link even if `#site` does not exist yet (the link is valid, but the user arrives at the top of the page instead of the correct section) and since this is not a regular link, it's not a problem with the checks.

For simplicity, I chose the second option. Just wanted to leave a note in case you check the link.

Reply:
Ah, maybe it was the comment (`//`) that made you wonder. It's a code snippet, so the comment is expected here. No need to remove it later.